static/js/admin/src/EditFilenameForm.js

```diff
@@ -0,0 +1,53 @@
+define('EditFilenameForm', [ 'jquery', 'jquery-ui', 'underscore', 'RemoteStateChangeMonitor', 'tpl!../templates/admin/editFilenameForm', 
+		'ModalLoadingOverlay', 'AbstractFileUploadForm', 'AlertHandler'], 
+		function($, ui, _, RemoteStateChangeMonitor, editFilenameForm, ModalLoadingOverlay, AbstractFileUploadForm) {
```

Comment:
I left this ref alone for now, since I patterned this class after EditLabelForm. Just thought maybe we should keep those consistent.